ladybug_comfort/wbgt.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+"""Utility functions for calculating the Wet Bulb Globe Temperature (WBGT)."""
+from __future__ import division
+
+import math
+
+
+def wbgt_outdoors(ta, mrt, ws, vp):
+    """
+    The wet-bulb globe temperature (WBGT) is a type of apparent temperature,
+    and is widely used heat stress index globally (ISO 7243). It is used to
```

Comment:
We usually prefer to have the docstrings formatted with a single concise line at the top. Then, a line break after which you can include whatever other information you want.